engine/crates/engine-v2/src/execution/header_rule.rs

```diff
@@ -0,0 +1,123 @@
+use std::{borrow::Cow, str::FromStr};
+
+use http::header;
+use schema::{HeaderRuleWalker, NameOrPatternRef};
+
+use crate::engine::RequestContext;
+
+static FORWARD_BLACKLIST: [header::HeaderName; 12] = [
+    header::ACCEPT,
+    header::ACCEPT_CHARSET,
+    header::ACCEPT_ENCODING,
+    header::ACCEPT_CHARSET,
+    header::ACCEPT_LANGUAGE,
```

Comment:
I can see why we'd want to get rid of most of the accepts, but language doesn't seem like something we'd care about?  Ofc not sure who would use it, just a thought

Reply:
hum you're right, ended up just copy pasting all ACCEPT